atomics/T1036/T1036.yaml

```diff
@@ -80,3 +80,57 @@ atomic_tests:
       cmd.exe /K %APPDATA%\taskhostw.exe
     cleanup_command: |
       del /Q /F %APPDATA%\taskhostw.exe
+
+- name: Masquerading - non-windows exe running as windows exe
+  description: |
+    Copies an exe, renames it as a windows exe, and launches it to masquerade as a real windows exe
+  
+  supported_platforms:
+    - windows
+  
+  input_arguments:
+    inputfile:
+      description: path of file to copy
+      type: path
+      default: $PathToAtomicsFolder\T1036\bin\t1036.exe
```

Comment:
Hi @bnt1006 , This all looks really good. My only question is, do you have this defined on your local system and not in the test?  Would you mind modifying/adding a new argument for path. That should resolve this.

Thank you!

Reply:
Thank you for reviewing @MHaggis! The $PathToAtomicsFolder variable, I believe, should be set on everyone's box according to the commit yesterday by clr2of8 "PathToAtomicsFolder Input Parameters auto-replaced with actual path." I'm happy to change it if that is still needed. Do you have a suggestion for finding the atomic's folder path on everyone's machine?

Reply:
yes, you're right! Spoke with clr2of8 on this. Thank you!